### PR TITLE
support group coordinator standalone

### PR DESCRIPTION
**Pull request by fu-turer**

### CODE REVIEW

File: pkg/kafsar/group_coordinator_standalone_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 var (
 	clientId         = "consumer-a1e12365-ddfa-43fc-826e-9661fb54c274-1"
 	groupId          = "test-group-id"
-	memberId         = "test-member-id"
+	memberId         = ""
```
Comment (Shoothzj) [inline pkg/kafsar/group_coordinator_standalone_test.go:29]: I suggest create two test instead of change value

Comment (fu-turer) [reply pkg/kafsar/group_coordinator_standalone_test.go:29]: it should be empty, when join group fist time

File: pkg/kafsar/group_coordinator_standalone_test.go [hunk 2]
```diff
@@ -24,93 +24,137 @@ import (
 )
 
 var (
-	clientId         = "reader-a1e12365-ddfa-43fc-826e-9661fb54c274-1"
+	clientId         = "consumer-a1e12365-ddfa-43fc-826e-9661fb54c274-1"
```
Comment (Shoothzj) [inline pkg/kafsar/group_coordinator_standalone_test.go:27]: unnecessary change

File: pkg/kafsar/group_coordinator_standalone_test.go [hunk 3]
```diff
@@ -24,93 +24,137 @@ import (
...
 	groupId          = "test-group-id"
-	memberId         = "test-member-id"
+	memberId         = ""
 	sessionTimeoutMs = 30000
-	protocolType     = "reader"
+	protocolType     = "consumer"
```
Comment (Shoothzj) [inline pkg/kafsar/group_coordinator_standalone_test.go:31]: unnecessary change

**Review by tyluffy — APPROVED** [review]

**Review by goflutterjava — APPROVED** [review]
